modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientKeyValueView.java

```diff
@@ -229,6 +235,11 @@ public CompletableFuture<Void> putAllAsync(@Nullable Transaction tx, Map<K, V> p
             return nullCompletedFuture();
         }
 
+        for (Entry<K, V> e : pairs.entrySet()) {
+            Objects.requireNonNull(e.getKey());
```

Comment:
should we add some message to help a user identify a problem?

Reply:
We should. But all assertions in this classes and their siblings do not return any message.